src/ClientBuilder.php

```diff
@@ -121,11 +123,13 @@ public function __construct(array $options = [])
     {
         $this->options = new Options($options);
 
-        if (null !== $this->options->getIntegrations()) {
+        if ($this->options->getDefaultIntegrations()) {
```

Comment:
Instead of having a pair of `if`/`else` maybe we can improve the readability of the code a bit by editing it like the following one:

```php
$this->integrations = $this->options->getIntegrations();

if ($this->options->getDefaultIntegrations()) {
    // Note: does it matter if default integrations are registered before the custom ones?
    $this->integrations = array_merge($this->integrations, [...]);
}
```

Note that you should check it still covers the cases you're trying to fix

Reply:
Changed, it's better to be initialized after the default ones in case your custom integration wants to change something.